build-system/tasks/release/index.js

```diff
@@ -13,9 +13,9 @@ let ExperimentConfigDef;
 
 /**
  * @typedef {{
- *  experimentA: ExperimentConfigDef,
- *  experimentB: ExperimentConfigDef,
- *  experimentC: ExperimentConfigDef,
+ *  experimentA: ExperimentConfigDef | {},
```

Comment:
Could this be `experimentA?: ExperimentConfigDef`?

Reply:
These are not the same types though, and experiments are either configured (all 5 fields) or are an empty object (`{}`) - by having this `|` type rule here I can later cast to this type and have the type checker figure out that if one field is set, all other fields are set